### PR TITLE
Closes #157. Optimized Images and a styling fix

### LINKED ISSUE #157: Image size on website
_From @bardware on September 7, 2018 13:46_

Hi,

I just stumble over your website and see you embed logos of languages that are 400 kByte in size. That's a lot. It looks to me it's a png embedded in svg. Maybe you can resize it.
grafik

_Copied from original issue: Unibeautify/unibeautify#178_

### PR DESCRIPTION
**Pull request by muuvmuuv**
- Optimized all images based on #157 
- Changed the default cursor type on hover for detail elements

### CODE REVIEW

**Comment by muuvmuuv** [comment]
@Glavin001 could you help me here with resolving his conflict, I don't know how to do this. Normally I just merge the master into mine, but this time this did not work...

**Comment by Glavin001** [comment]
@muuvmuuv I've resolved the merge conflicts.

You should now run `yarn` in both the project root and `website/` directory. This will update and finish resolving merge conflict problems in `yarn.lock`.

**Comment by muuvmuuv** [comment]
@Glavin001 Thanks! So, can we merge now?

**Comment by Glavin001** [comment]
@muuvmuuv Something is still wrong. Merge conflicts. Can you pull the latest code, it'll tell you there are merge conflicts, then run `yarn` again. Strange 😕 .

File: website/pages/en/index.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ const LanguagesShowcase = () => {
       <div className="logos">
           {showcase}
           <a href={"/docs/languages.html"}>
-              <img src={imgUrl("font-awesome/plus-square-regular.svg")} title={"And more..."}/>
+              <img src={imgUrl("languages/more.png")} title={"And more..."}/>
```
Comment (Glavin001) [inline website/pages/en/index.js]: Thanks for improving this image! 😃 🎉

File: website/i18n/en.json [hunk 2]
```diff
@@ -4,277 +4,523 @@
...
-    "S": "S",
-    "U": "U",
-    "W": "W",
-    "G": "G",
-    "H": "H",
-    "L": "L",
-    "T": "T",
-    "V": "V",
-    "X": "X"
+    "docs": {
```
Comment (Glavin001) [inline website/i18n/en.json]: What happened here?

**Comment by Glavin001** [comment]
Awesome work. Thanks for contributing @muuvmuuv !

**Pull request merged**